libsolidity/analysis/PostTypeChecker.cpp

```diff
@@ -131,6 +131,16 @@ void PostTypeChecker::endVisit(ModifierInvocation const& _modifierInvocation)
 	callEndVisit(_modifierInvocation);
 }
 
+bool PostTypeChecker::visit(FunctionDefinition const& _functionDefinition)
```

Comment:
Why are these needed?

Reply:
it's a left-over from an earlier attempt that i already deleted & squashed away.